Formula/anime-downloader.rb

```diff
@@ -0,0 +1,83 @@
+class AnimeDownloader < Formula
+  include Language::Python::Virtualenv
+
+  desc "Simple but powerful anime downloader and streamer"
+  homepage "https://github.com/vn-ki/anime-downloader"
+  url "https://github.com/vn-ki/anime-downloader/archive/3.6.3.tar.gz"
+  sha256 "5908263efebd535089b9a856ae3855171a30accec49b827ff4e71245d5115757"
+  head "https://github.com/vn-ki/anime-downloader.git"
+
+  bottle :unneeded
```

Comment:
This is inappropriate.
I cannot provide a clear rule, but generally you shouldn't use `bottle: unneeded` unless you are asked by Homebrew members.

Reply:
Anything that just installs a file/universal binary (like most php and java based projects) and doesn't need any real install can be `bottle :unneeded`. Any other new formula should not have a bottle block because the bot generates them.

Reply:
This has been removed, thanks.